team-manager/src/test/unit/java/ua/knu/knudev/teammanager/service/DepartmentServiceTest.java

```diff
@@ -151,4 +157,163 @@ void should_throwDepartmentException_When_DepartmentDoesNotExistOnGet() {
 
         verify(departmentRepository, times(1)).findById(TEST_DEPARTMENT_ID);
     }
+
+    @Test
+    @DisplayName("Should create department when given valid data")
+    public void should_createDepartment_When_givenValidData() {
+        SpecialtyCreationDto specialtyCreationDto1 = new SpecialtyCreationDto(14.378, "Computer Engineering");
+        SpecialtyCreationDto specialtyCreationDto2 = new SpecialtyCreationDto(14.378, "Radio technics");
+        SpecialtyCreationDto specialtyCreationDto3 = new SpecialtyCreationDto(14.378, "History");
```

Comment:
use here method getTestSpecialty(name)

Reply:
no? because i need here creationDto not Domain

Reply:
Yes. Its actually better to just instantiate via `new` keyword